rumqttd/rumqttd.toml

```diff
@@ -5,11 +5,13 @@ id = 0
 # latencies of other connection. Not a problem with preempting runtimes
 [router]
 id = 0
-instant_ack = true
+max_connections = 10010
+max_outgoing_packet_count = 200
 max_segment_size = 104857600
 max_segment_count = 10
-max_read_len = 10240
-max_connections = 10010
+    [router.custom_segment.'/tenants/+/+/+/actions']
```

Comment:
do we want to also limit the size of segment when subscription is on filter `/tenants/+/+/+/actions`

Reply:
Yep, smaller segments and less in number for stream that contains device actions